src/Tools/dotnet-watch/test/TestProjects/KitchenSink/.net/objDebug/net5.0/.NETCoreApp,Version=v5.0.AssemblyAttributes.cs

```diff
@@ -0,0 +1,4 @@
+// <autogenerated />
```

Comment:
What requires checking in auto-generated files❔

Reply:
Removed